### ISSUE TITLE
Query form looks in uploads instead of database

### ISSUE THREAD

**Issue by e-koch** [camelot-project/frontend#26]
```
IOError: [Errno 2] No such file or directory: u'uploads/merged_table.ipac'
```

**Comment by scog1234** [comment]
UPLOAD_FOLDER needs changing to DATABASE_FOLDER on this line:
     table = Table.read(os.path.join(app.config['UPLOAD_FOLDER'], filename), format='ascii.ipac')

**Comment by e-koch** [comment]
Addressed in #27
